client/src/components/feddback-comment.js

```diff
@@ -0,0 +1,45 @@
+import React, { Component } from 'react';
+import { Container } from 'react-bootstrap'
+import '../style/feedback.css'
+
+class FeedbackComment extends Component {
+
+
+    render() {
+
+        return (
+
+            <Container className="feedbackUserContainer">
```

Comment:
We'd rather use simplier classNames, not like `feedbackUserContainer`

Reply:
Aldo see lines 14, 22, 32. See [this](https://www.sitepoint.com/title-css-simple-approach-css-class-naming/) for styles integration and class naming in our case.

Reply:
Read class Name and you understand what he do. It`s simple.